gallery/src/main.scss

```diff
@@ -9,6 +9,16 @@
 @import '~@sonatype/react-shared-components/scss-shared/nx-container-helpers';
 @import '~@sonatype/react-shared-components/scss-shared/nx-flexbox-helpers';
 
+.nx-page.gallery-page {
+  .nx-page-title {
+    padding-left: $nx-spacing-md;
+  }
+  .nx-page-main {
+    padding-left: $nx-spacing-xl;
+    padding-right: $nx-spacing-xl;
```

Comment:
Why these custom styles?  The extra padding on `nx-page-title` in particular seems not to match the mockups.

Reply:
> Why these custom styles? The extra padding on `nx-page-title` in particular seems not to match the mockups.

I moved the l/r padding to _page-layout where it belongs. https://github.com/sonatype/sonatype-react-shared-components/pull/40/commits/de3ff806abcf41d01f72e479e8e0b78e86531390